internal/statemachine/snap_states.go

```diff
@@ -33,6 +33,7 @@ func (stateMachine *StateMachine) prepareImage() error {
 	imageOpts.Preseed = snapStateMachine.Opts.Preseed
 	imageOpts.PreseedSignKey = snapStateMachine.Opts.PreseedSignKey
 	imageOpts.AppArmorKernelFeaturesDir = snapStateMachine.Opts.AppArmorKernelFeaturesDir
+	imageOpts.SeedManifestPath = filepath.Join(stateMachine.commonFlags.OutputDir, "seed.manifest")
```

Comment:
If using the `--workdir` flag instead of the `--output` flag this variable won't be assigned yet. This is actually something I've fixed in the feature branch for redesigning classic images, so perhaps we can just ignore this for now.